src/recipeRepository.js

```diff
@@ -1,4 +1,4 @@
-class Cookbook {
+class RecipeRepository {
```

Comment:
Thank you for going through and updating this to have a more appropriate name for this class!